Dockerfile.template

```diff
@@ -1,46 +1,28 @@
-FROM resin/%%RESIN_MACHINE_NAME%%-debian:jessie
+FROM balenalib/%%BALENA_MACHINE_NAME%%:buster
 
-RUN apt-get update && apt-get install -y --no-install-recommends \
-	python-pip \
-	python-dev \
+RUN apt update && apt install -y --no-install-recommends \
+	# build-essential \
+	ffmpeg \
 	git \
-	build-essential \
-	libav-tools \
-	avrdude \
-	curl \
-	subversion \
-	libjpeg8-dev \
-	imagemagick \
-	libav-tools \
-	cmake \
-	haproxy \
-  && apt-get clean \
+	python3-dev \
+	python3-picamera \
+	python3-pip \
+	python3-requests \
+  && apt autoremove \
+  && apt clean \
   && rm -rf /tmp/* /var/tmp/*  \
   && rm -rf /var/lib/apt/lists/*
 
-# Enable haproxy
-RUN echo 'ENABLED=1' >> /etc/default/haproxy
-
-# Deploy haproxy config
-COPY ./Dockerbin/haproxy.cfg /etc/haproxy/haproxy.cfg
-
 # Move to app dir
 WORKDIR /usr/src/app
 
-# Install Octoprint from source
-RUN git clone https://github.com/foosel/OctoPrint.git ./octoprint \
-  && cd /usr/src/app/octoprint && pip install --upgrade pip && pip install setuptools \
-  && cd /usr/src/app/octoprint && pip install -r requirements.txt && python setup.py install \
-  && cd /usr/src/app/octoprint && pip install https://github.com/BillyBlaze/OctoPrint-TouchUI/archive/master.zip \
-	&& cd /usr/src/app/octoprint && pip install https://github.com/marian42/octoprint-preheat/archive/master.zip \
-	&& cd /usr/src/app/octoprint && pip install "https://github.com/OctoPrint/OctoPrint-DisplayProgress/archive/master.zip" \
-	&& cd /usr/src/app/octoprint && pip install https://github.com/malnvenshorn/OctoPrint-CostEstimation/archive/master.zip
+# Install OctoPrint from source
+RUN git clone https://github.com/foosel/OctoPrint.git \
```

Comment:
Needs to be `https://github.com/OctoPrint/OctoPrint`

Reply:
Ok, that may be more correct nowadays. It seems that the repo was moved at some point. The foosel/OctoPrint does properly redirect, however, so this still works which I why I never changed it.

Reply:
Ok, this is done.

Reply:
It was only just moved 2 weeks ago, so it was easy to miss!